cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -54,6 +54,15 @@
 PCLUSTER_LAMBDA_PREFIX = "pcluster-"
 
 
+def _get_resource_combination_name(resource_name_1, resource_name_2, partial_length, hash_length):
```

Comment:
Seems restricted  to just 2 resource names, can we use something like this?

```
def _get_resource_combination_name(*resource_names, partial_length=6, hash_length=16):
  combined_name = "".join(resource_names)
  hash_value = sha256(combined_name.encode()).hexdigest()[:hash_length]
  prefix = "-".join(name[:partial_length] for name in resource_names)
  return f"{prefix}-{hash_value}"
```

Reply:
That a brilliant point! Thanks Eddy!